src/include/sof/audio/component.h

```diff
@@ -155,7 +155,8 @@ enum comp_endpoint_type {
  /* \brief Type of component copy, which can be changed on runtime */
 enum comp_copy_type {
 	COMP_COPY_NORMAL = 0,
-	COMP_COPY_BLOCKING
+	COMP_COPY_BLOCKING,
+	COMP_COPY_ONE_SHOT
 };
```

Comment:
Exactly, my point here. :) Also add comma after COMP_COPY_ONE_SHOT for the same reason :)